dawg_python/dawgs.py

```diff
@@ -315,9 +346,52 @@ def iteritems(self, prefix=""):
 
         while completer.next():
             key, value = completer.key.split(self._payload_separator)
-            item = (key.decode('utf8'), a2b_base64(bytes(value))) # bytes() cast is a python 2.6 fix
+            # bytes() cast is a python 2.6 fix
+            item = (key.decode('utf8'), a2b_base64(bytes(value)))
             yield item
 
+    def edges(self, prefix=""):
```

Comment:
If we really want them to be the same, we could make them return `(str, True)` for terminal edges always, and just add an extra `edges_with_data()` method for dawgs that provide any kind of data storage. That actually seems most consistent to me. If you agree, I'll make that addition.